scripts/ldap/base_configure_ldap.py

```diff
@@ -0,0 +1,155 @@
+import sys
+import os
+import ldap
+import glob
+from shutil import copyfile
+from ldap.ldapobject import SimpleLDAPObject
+
+usage = "USAGE: python common.py [ldapadminpassword] [rootdnpassword] [usedefaultpassword] [skipssl] [forceclean]"
+totalArguments = len(sys.argv)
+
+defaultpasswd=False
+usessl=True
+forceclean=False
+LDAPADMINPASS=None
+ROOTDNPASSWORD=None
+defaultpasswds="ldapadmin"
+
+if totalArguments <= 1 :
+    print('Too few arguments.' , usage)
+    sys.exit()
+if totalArguments > 1 :
+    LDAPADMINPASS =  sys.argv[1]
+if totalArguments > 2 :
+    ROOTDNPASSWORD = sys.argv[2]
+if totalArguments > 3 and sys.argv[3] == usedefaultpassword :
+    defaultpasswd = True
+if totalArguments > 4 and sys.argv[4] == skipssl :
+    usessl = False
+if totalArguments > 5 and sys.argv[5] == forceclean :
+    forceclean = True
+
+INSTALLDIR="/opt/seagate/cortx/s3/install/ldap"
+#cleanup
+# Removing schemas
+try:
+    os.remove('/etc/openldap/slapd.d/cn=config/cn=schema/cn={1}s3user.ldif')
+except:
+    print('Error while deleting s3user.ldif file')
```

Comment:
In all print messages please print filename with full path

Reply:
Fixed